Update set-up-sso-saml-2.0.md (#2701)
fishtown-analytics/docs.getdbt.com: website/docs/docs/collaborate/manage-access/set-up-sso-saml-2.0.md

### Configure the Okta application

> The following steps assume your dbt Cloud instance is running at `https://cloud.getdbt.com`. If your deployment is running at a different url, then substitute ` cloud.getdbt.com` for the url of your instance.

Reviewer: sugg adding link to regions page. also might be worth creating a snippet for this content since i noticed it's repeated in many sections. happy to help with that!

```suggestion
The following steps assume your dbt Cloud deployment uses`https://cloud.getdbt.com`. If your [deployment](/docs/deploy/regions-ip-addresses) uses a different URL, substitute ` cloud.getdbt.com` with the URL of your instance. 
```

To complete this section, you will need to create a login slug. This slug controls the URL where users on your account
can log into your application. Login slugs are typically the lowercased name of your organization
separated with dashes. For example, the login slug for dbt Labs would be `dbt-labs`.
Login slugs must be unique across all dbt Cloud accounts, so pick a slug that uniquely identifies your company.

> 

Reviewer: ✅ Use: your-company-name
❌ Avoid: Your-Company-Name

> 

Reviewer: recommend adding below as guidance

✅ Use: your-company-name
❌ Avoid: Your-Company-Name

### Configure the Google application

> The following steps assume your dbt Cloud instance is running at `https://cloud.getdbt.com`. If your deployment is running at a different url, then substitute ` cloud.getdbt.com` for the url of your instance.

Reviewer: ```suggestion
The following steps assume your dbt Cloud instance is running at `https://cloud.getdbt.com`. If your [deployment](/docs/deploy/regions-ip-addresses) uses a different URL, substitute ` cloud.getdbt.com` with the URL of your instance.
```

4. Enter the following values on the **Service Provider Details** window:
   - **ACS URL**: `https://cloud.getdbt.com/complete/saml`
   - **Audience URI (SP Entity ID)**: `https://cloud.getdbt.com/`
   - **Start URL**: `<login slug`

> 5. Check the **Signed response** checkbox.

Reviewer: changing word to adhere to [style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/mwong-styleguide-add/contributing/content-style-guide.md)

```suggestion
5. Select the **Signed response** checkbox.
```

6. The default **Name ID** is the primary email. Multi-value input is not supported.

> 7. Use the **Attribute mapping** page to map your organization's Google Directory Attributes to the format that

Reviewer: non blocking but wanted to ask if there were any examples of the format? or is that clear in cloud?

Author: It should be clear :)

| `Last name`    | Unspecified | `last_name`          | The user's last name.     |
| `Primary email`| Unspecified | `email`              |  The user's email address. |

> 9. To use role-based access control in dbt Cloud,enter the groups in the **Group membership** field during configuration:

Reviewer: adding rbac doc link and added spacing after comman

```suggestion
9. To use [role-based access control](/docs/collaborate/manage-access/about-access#role-based-access-control) in dbt Cloud,  enter the groups in the **Group membership** field during configuration:
```

### Create Azure AD Enterprise application

> The following steps assume your dbt Cloud instance is running at `https://cloud.getdbt.com`. If your deployment is running at a different url, then substitute ` cloud.getdbt.com` for the url of your instance.

Reviewer: ```suggestion
The following steps assume your dbt Cloud instance is running at `https://cloud.getdbt.com`. If your [deployment](/docs/deploy/regions-ip-addresses) uses a different URL, substitute ` cloud.getdbt.com` with the URL of your instance.
```

### Configure the OneLogin application

> The following steps assume your dbt Cloud instance is running at `https://cloud.getdbt.com`. If your deployment is running at a different url, then substitute ` cloud.getdbt.com` for the url of your instance.

Reviewer: ```suggestion
The following steps assume your dbt Cloud instance is running at `https://cloud.getdbt.com`. If your [deployment](/docs/deploy/regions-ip-addresses) uses a different URL, substitute ` cloud.getdbt.com` with the URL of your instance.
```

### Configure SAML settings

3. Under the **Configuration tab**, input the following values:

   - **RelayState:** `<login slug>`
   - **Audience (EntityID):** https://cloud.getdbt.com/
   - **ACS (Consumer) URL Validator:** https://cloud.getdbt.com/complete/saml
   - **ACS (Consumer) URL:** https://cloud.getdbt.com/complete/saml

> 4. Next, go to the **Parameters tab**. You must have a parameter for the Email, First Name, and Last Name attributes and include all parameters in the SAML assertions. When you add the custom parameters, make sure you check the **Include in SAML assertion** checkbox.

Reviewer: to adhere to style guide

```suggestion
4. Next, go to the **Parameters tab**. You must have a parameter for the Email, First Name, and Last Name attributes and include all parameters in the SAML assertions. When you add the custom parameters, make sure you select the **Include in SAML assertion** checkbox.
```
